packages/server/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "eslint 'src/**/*.{ts,tsx}'",
     "build": "webpack --config webpack.prod.js",
+    "build-dev": "webpack --config webpack.dev.js",
     "prestart": "webpack --config webpack.dev.js",
```

Comment:
is this one redundant?

Reply:
Yeah it's a good point - so the `prestart` is nice in that running `yarn start` will automatically run `prestart` first, so it's kind of special. I guess we could do

```
"prestart": "yarn build-dev",
```